src/main/java/org/geppetto/core/manager/IGeppettoManager.java

```diff
@@ -66,5 +67,9 @@
 	 */
 	Scope getScope();
 
+	void setSimulationListener(IGeppettoManagerCallbackListener listener);
 
+	void setLogin(boolean b);
+	
+	boolean isLogin();
```

Comment:
Can we refactor these out in favor of detecting the presence of the persistence bundle?

Something like this can be done to test that on the frontend:

`!AuthServiceCreator.getService().isDefault()`